asciidocs/setup.adoc

```diff
@@ -2,19 +2,19 @@
 
 There are two main ways to get started with Seqera's Nextflow training course.
 
-The first is to install the requirements locally (section 1.1), which is best if you are already familiar with Git and Docker, or working offline.
+The first is to install the requirements locally (<<Local installation>>), which is best if you are already familiar with Git and Docker, or working offline.
 
-The second is to use Gitpod (section 1.2), which is best for first-timers as this platform contains all the programs and data required.
-Simply click the link and login via your GitHub account to start the tutorial.
+The second is to use Gitpod (<<Gitpod>>), which is best for first-timers as this platform contains all the programs and data required.
```

Comment:
Perhaps best to just link the section here since it appears a bit repetitive like `Gitpod (Gitpod)`? 

```suggestion
The second is to use <<Gitpod>>, which is best for first-timers as this platform contains all the programs and data required.
```